environments.json

```diff
@@ -5033,6 +5033,14 @@
     "release": "2024-05-03",
     "obsolete": true
   },
+  "rhino1_8_0": {
+    "full": "Rhino 1.8.0",
+    "short": "Rhino 1.8.0",
+    "family": "Rhino",
+    "platformtype": "engine",
+    "release": "2025-01-03",
+    "obsolete": true
```

Comment:
Since this is the latest release, shouldn't it be `"obsolete": false,`?

Reply:
See: https://github.com/compat-table/compat-table/issues/1687#issuecomment-729878415

Reply:
Ok! Delighted to set that to false